<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerDlg</class>
 <widget class="QDialog" name="registerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/File/E:/SE_work/Qt_project/OnlibraryV1/images/图标.png</normaloff>:/File/E:/SE_work/Qt_project/OnlibraryV1/images/图标.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit { 
    border: 1px solid rgb(41, 57, 85);
    background: white;
    selection-background-color: darkgray;
    font-size: 16px ;
}

QLineEdit:hover {
	border: 2px solid blue;
}

QLineEdit[echoMode=&quot;2&quot;] {
      lineedit-password-character: 42;
}

QPushButton{
	border-radius:6px;
	background-color: #1390C5;
	min-width:80px;
}

QPushButton:pressed {
	background-color: #105D7E;
}

QPushButton:hover {
	background-color: #2BA0F3;
}

color: rgb(211, 211, 211);

</string>
  </property>
  <widget class="QPushButton" name="commitBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ruid">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_rpwd">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_rname">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_raddress">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_rtel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
      </property>
      <property name="text">
       <string>邮箱:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
      </property>
      <property name="text">
       <string>密码:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
      </property>
      <property name="text">
       <string>姓名:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
      </property>
      <property name="text">
       <string>邮寄地址:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
      </property>
      <property name="text">
       <string>联系方式:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>registerDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
